packages/emptystate/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../base-tsconfig.json",
```

Comment:
you should just be able to use the tsconfig that is already in the root. this pr that i'm working on is doing just that

https://github.com/pluralsight/design-system/pull/1141

@KaidenR do you want to sync up maybe 😉

Reply:
Oh I didn't realize you were already working on converting something to TS 🤦  
Should I just wait for that one to wrap up and then look at how that affects this PR?

Reply:
sorry 😬  - i'm going to switch gears and try to wrap up my pr today. i'll be asking you for a review...you've gotten yourself into